__tests__/app.test.js

```diff
@@ -160,4 +160,79 @@ describe('/api/articles/:article_id/comments', () => {
             expect(response.body.msg).toBe('Bad request');
         });
     });
+});
+
+    describe('/api/articles/:article_id/comments', () => {
+        test('Status 201: Responds with object containing the newly posted comment', () => {
```

Comment:
optional: we could add a test that posts a comment if the necessary properties are there and ignores any additional unnecessary properties